plugin/cypress/e2e/duckduckgo.ts

```diff
@@ -0,0 +1,13 @@
+import { When, Then } from "@badeball/cypress-cucumber-preprocessor";
+
+When("I visit duckduckgo.com", () => {
+  cy.visit("https://www.duckduckgo.com");
+});
+
+Then("I should see a search bar", () => {
+  cy.get("input").should(
```

Comment:
The test does not work for me, I have fixed it adding id to input 
```
cy.get('input#searchbox_input')
```

Reply:
For Jenkins integration it's better to have it  failing for now. It's just dummy test, will be removed in next PR coming soon.